### PR TITLE
Only attachments editable fixes

### PR DESCRIPTION
**Pull request by heeboA**
- Fix hakemus not opening for editing
- Fix fields with no answer in backend being editable since the non-existing answer couldn't obviously be flagged as uneditable.

### CODE REVIEW

File: src/clj/ataru/hakija/hakija_application_service.clj [hunk 1]
```diff
@@ -77,28 +77,68 @@
...
+  {:key       id
+   :fieldType fieldType
+   :label     label
+   :value     ""})
+
+(defn- filter-questions-without-answers
+  [answers-by-key form-fields]
+  (filter (fn [answer]
+            (and (not (some #{(keyword (:id answer))} (keys answers-by-key)))
+                 (some #{(:fieldType answer)} ["textField"
```
Comment (starvinraven) [inline src/clj/ataru/hakija/hakija_application_service.clj]: I'm a bit worried about these kinds of "magic lists" that can easily be left forgotten when e.g. creating new types of fields in the future. Could this either check for `:fieldClass` with value `"formField"` or if not, be extracted somehow?

File: src/clj/ataru/hakija/hakija_application_service.clj [hunk 2]
```diff
@@ -77,28 +77,68 @@
...
+                                               "dropdown"
+                                               "multipleChoice"
+                                               "singleChoice"
+                                               "attachment"
+                                               "hakukohteet"]) ; no container fields!
+                 (not (:followup? answer)) ; make sure followup answers don't show when parent not selected
+                 (not (:exclude-from-answers answer)))) form-fields))
+
+(defn- get-questions-without-answers
+  "This function serves to get dummy answers and their editability (mainly for 10 day crage period of attachments
```
Comment (starvinraven) [inline src/clj/ataru/hakija/hakija_application_service.clj]: "grace" period :)

File: src/clj/ataru/hakija/hakija_application_service.clj (hunk 1 above)
Comment (heeboA) [reply src/clj/ataru/hakija/hakija_application_service.clj]: I was thinking about the same, i'll extract it..

File: src/clj/ataru/hakija/hakija_application_service.clj (hunk 2 above)
Comment (heeboA) [reply src/clj/ataru/hakija/hakija_application_service.clj]: Nicholas Grace..

**Pull request merged**